src/CrossReferences.jl

```diff
@@ -116,11 +116,37 @@ function docsxref(link::Markdown.Link, meta, page, doc)
     # Parse the link text and find current module.
     local code = link.text[1].code
     local keyword = Symbol(strip(code))
-    local ex = haskey(Docs.keywords, keyword) ? QuoteNode(keyword) : parse(code)
+    local ex
+    if haskey(Docs.keywords, keyword)
+        ex = QuoteNode(keyword)
+    else
+        try
+            ex = parse(code)
+        catch err
+            !isa(err, ParseError) && rethrow(err)
+            Utilities.warn(page.source, "Unable to parse the reference '[`$code`](@ref)'.")
+            return
```

Comment:
Codecov is saying this block isn't being tested. Can we add a malformed `code` to cover it?